cmd/storage-rest-server.go

```diff
@@ -284,9 +285,9 @@ func (s *storageRESTServer) ReadAllHandler(w http.ResponseWriter, r *http.Reques
 		s.writeErrorResponse(w, err)
 		return
 	}
-	w.Header().Set(xhttp.ContentLength, strconv.Itoa(len(buf)))
```

Comment:
Maybe we should set this as an alternative header?

Reply:
I thought since we will send the content as transfer-encoding: chunked - this won't be needed.